### ISSUE TITLE
Add claim option for VG

### ISSUE THREAD

**Issue by NejcZdovc** [brave/brave-browser#8354]
Right now we don't have a mechanism to transfer virtual grants from ad earnings to uphold. 

We need to check for claim on two events. Once user connect KYC uphold account and once user becomes KYC. 

We first need to call old API just once (so that we record paymentID) and then new api `v1/suggestions/claim`.

**Comment by serp777** [comment]
Just wondering if there were any updates on this or if any assistance would be useful?

**Comment by NejcZdovc** [comment]
@serp777 I have already PR here that is complete https://github.com/brave/brave-core/pull/4708. But if you are interested we have a lot of things going on with rewards. You can check issues here https://github.com/brave/brave-browser/projects/5

**Comment by LaurenWags** [comment]
Needs new 1.5.x RC build to test this, should be available in 1.5.112 and above.

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 1.5.112 Chromium: 80.0.3987.132 (Official Build) (64-bit)
-- | --
Revision | fcea73228632975e052eb90fcf6cd1752d3b42b4-refs/branch-heads/3987@{#974}
OS | macOS Version 10.14.6 (Build 18G3020)

* Verified 5 scenarios from https://github.com/brave/brave-core/pull/4708#issuecomment-597287084

**PASS** New User Scenario
```
[ RESPONSE - OnTransferAnonToExternalWallet ]
> time: 1584468243
> result: Success
```

```
[ RESPONSE - DeleteTokens ]
> time: 1584468244
> result: Success
```

**PASS** Upgrade Scenario 1 - partially spent ad grant
```
[ RESPONSE - OnTransferAnonToExternalWallet ]
> time: 1584469722
> result: Success
```

```
[ RESPONSE - DeleteTokens ]
> time: 1584469902
> result: Success
```

**PASS** Upgrade Scenario 2 - full ad grant (not partially spent prior to upgrade)

```
[ RESPONSE - OnTransferAnonToExternalWallet ]
> time: 1584470347
> result: Success
```

```
[ RESPONSE - DeleteTokens ]
> time: 1584470532
> result: Success
```

**PASS** PR test plan 1

```
[ RESPONSE - OnTransferAnonToExternalWallet ]
> time: 1584470927
> result: Success
```

```
[ RESPONSE - DeleteTokens ]
> time: 1584470928
> result: Success
```

**PASS** PR test plan 2
```
[ RESPONSE - OnTransferAnonToExternalWallet ]
> time: 1584471983
> result: Success
```

```
[ RESPONSE - DeleteTokens ]
> time: 1584472036
> result: Success
```
